shared/gh/js/constants/gh.constants.js

```diff
@@ -42,4 +42,60 @@ define(['exports'], function(exports) {
             'error': 'This is a Timetable error, if you still see this after reloading a Timetable in your browser please contact <a href="mailto:help@timetable.cam.ac.uk" title="Contact support">support</a>'
         }
     };
+
+    var templates = exports.templates = {
```

Comment:
Can you just add a line of comment above this block (and the messaging block as well) to briefly explain why it's here?